presto-common/src/main/java/com/facebook/presto/common/Page.java

```diff
@@ -39,6 +40,10 @@
     public static final int INSTANCE_SIZE = ClassLayout.parseClass(Page.class).instanceSize();
     private static final Block[] EMPTY_BLOCKS = new Block[0];
 
+    public static long getInstanceSizeInBytes(int blockCount)
```

Comment:
Should we also update the function `updateRetainedSize` to use this new function to get the instance size?